woocommerce/woocommerce-gutenberg-products-block
# Fix/2630 adjust twenty twenty font size (reopened)

### Purpose of this PR

When using the Twenty Twenty theme, the headlines of the _Active Product Filters_, _Filter Products by Price_, _Filter Products by Attribute_ & _Filter Products by Stock_ blocks are huge. In addition, the font size in the editor is different from the one on the frontend. This PR aims to make create a consistent font size both in the editor and the frontend as well as making the font size smaller.

Fixes #2630 

#### Accessibility

n/a

### Screenshots

<table>
<tr>
<td>Editor before:
<br><br>

![#2630-editor-before](https://user-images.githubusercontent.com/3323310/128177253-29760c2e-05df-4f65-800d-071bc33534f5.png)
</td>
<td>Editor after:
<br><br>

![#2630-editor-after](https://user-images.githubusercontent.com/3323310/128177189-2c749dcb-dae8-429b-8f81-20da3ec6b1b1.png)
</td>
</tr>
</table>

<table>
<tr>
<td>Frontend before:
<br><br>

![#2630-frontend-before](https://user-images.githubusercontent.com/3323310/128176355-e2680e23-1d30-4f7b-8ca9-35e04119cdf5.png)
</td>
<td>Frontend after:
<br><br>

![#2630-frontend-after](https://user-images.githubusercontent.com/3323310/128176352-884eff43-ec7c-4c38-9d79-618528635173.png)
</td>
</tr>
</table>

### How to test the changes in this Pull Request:

1. Install and activate the WooCommerce plugin
2. Install and activate this plugin
3. Create a new page
4. Add the column block (1/3 + 2/3) to the created page
5. Add the _Active Product Filters_, _Filter Products by Price_, _Filter Products by Attribute_ & _Filter Products by Stock_ blocks to the 1/3 column
6. Add the _All Products_ & _Products by Category_ blocks to the 2/3 column
7. Check the font sizes of the elements in the editor
8. Check the font sizes of the elements in the frontend
9. Check out this PR
10. Check the font sizes of the elements in the editor again
11. Check the font sizes of the elements in the frontend again

### Changelog

> Twenty Twenty: Adjust font styles for filter and product grid blocks
